pages/Product/components/Header/components/Shipping/index.jsx

```diff
@@ -18,11 +17,11 @@ import styles from './style';
  * @return {JSX}
  */
 const Shipping = ({ shipping }) => (
-  <PlaceholderLabel className={styles.placeholder} ready={(shipping !== null)}>
-    {shipping && (
+  <Fragment>
```

Comment:
IMO this is not an improvement.
Whenever there is a shipping rate available it is not in sync with the other placeholders anymore.
I recorded a video and attached it to the ticket. It doesn't look nice.